lib/Template.js

```diff
@@ -26,36 +43,63 @@ const stringifyIdSortPredicate = (a, b) => {
 	return 0;
 };
 
+//TODO: Change type to Module when import works
+//https://github.com/Microsoft/TypeScript/issues/23375
+/**
+ * @param {Module} module the module to compare against
+ * @return {boolean} return true if module.id is equal to type "number"
+ */
 const moduleIdIsNumber = module => {
 	return typeof module.id === "number";
 };
 
-module.exports = class Template {
+class Template {
+	/**
+	 *
+	 * @param {Function} fn - a runtime function (.runtime.js) "template"
```

Comment:
I generally don't like using `Function` but it seems appropriate in this case 😔

Reply:
agreed